### PR TITLE
Implement scale argument for Arrow object

### PR DESCRIPTION
**Pull request by tomsal**
### Overview

Add a simple way to scale the geometric object `Arrow`.

Resolves https://github.com/pyvista/pyvista/issues/670.

### CODE REVIEW

File: pyvista/utilities/geometric_objects.py [hunk 1]
```diff
@@ -219,6 +223,14 @@ def Arrow(start=(0.,0.,0.), direction=(1.,0.,0.), tip_length=0.25,
     arrow.SetShaftResolution(shaft_resolution)
     arrow.Update()
     surf = pyvista.PolyData(arrow.GetOutput())
+
+    if scale == 'auto':
+        scale = float(np.linalg.norm(direction))
+    if isinstance(scale, float) or isinstance(scale, int):
+        surf.points *= scale
+    elif scale is not None:
+        raise("Scale must be either float, int or 'auto'.")
```
Comment (akaszynski) [inline pyvista/utilities/geometric_objects.py:232]:
```suggestion
        raise TypeError("Scale must be either float, int or 'auto'.")
```

**Comment by akaszynski** [comment]
Could you also add the following within `test_geometric_objects.py`:

```python

@pytest.mark.parametrize('scale', [None, 2.0, 4, 'auto'])
def test_arrow():
    surf = pyvista.Arrow([0, 0, 0], [1, 1, 1], scale=scale)
    assert np.any(surf.points)
    assert np.any(surf.faces)

def test_arrow_raises_error():
    with pytest.raises(TypeError)
        surf = pyvista.Arrow([0, 0, 0], [1, 1, 1], scale='badarg')
```

It's important to have good code coverage when we add new features.  For example the line containing. `raise('...')` would have not been called correctly:

```
raise('...')
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-b610bac05e83> in <module>
----> 1 raise('woa')

TypeError: exceptions must derive from BaseException
```

File: pyvista/utilities/geometric_objects.py (hunk 1 above)
Comment (tomsal) [reply pyvista/utilities/geometric_objects.py]: Good point!

**Comment by tomsal** [comment]
Fixing the tests.

**Comment by akaszynski** [comment]
Thanks for incorporating the tests.  Looks like I had a few syntax errors in there!

**Pull request merged**